doc/api/child_process.md

```diff
@@ -1362,6 +1383,28 @@ then this will be `null`.
 `subprocess.stdout` is an alias for `subprocess.stdio[1]`. Both properties will
 refer to the same value.
 
+### subprocess.unref()
+<!-- YAML
+added: v0.7.0
+-->
+
+By default, the parent will wait for the detached child to exit. To prevent
+the parent from waiting for a given `subprocess`, use the `subprocess.unref()`
```

Comment:
Maybe `` `for a given `subprocess` to exit` ``?

Reply:
Change made